cmd/piper.go

```diff
@@ -41,16 +41,25 @@ type GeneralConfigOptions struct {
 	HookConfig           HookConfiguration
 }
 
-// HookConfiguration contains the configuration for supported hooks, so far only Sentry is supported.
+// HookConfiguration contains the configuration for supported hooks, so far Sentry and Splunk are supported.
 type HookConfiguration struct {
 	SentryConfig SentryConfiguration `json:"sentry,omitempty"`
+	SplunkConfig SplunkConfiguration `json:"splunk,omitempty"`
```

Comment:
What do you mean by one of each? It is possible that you have sentry and splunk activated in the same pipeline run.

Reply:
I understood that there are two teams that need a splunk integration. Could it be that we have two splunk hooks in the future?

Reply:
If so there would not be two SplunkHooks inside the HookConfiguration. We are thinking of haveing maybe two different indexes where the data will be sent, but this would be handled inside of the SplunkHook.